webpack.dev.js

```diff
@@ -0,0 +1,15 @@
+const path = require("path");
+const { merge } = require("webpack-merge");
+const common = require("./webpack.common.js");
+
+module.exports = merge(common, {
+  mode: "development",
+  devtool: "source-map",
```

Comment:
very very NIT, but is it needed to set `devtool` and `mode` here since it's being set in `common?` Mostly wondering if it's easier to understand if we're only setting params that definitely need to be set